.github/workflows/checks.yml

```diff
@@ -13,7 +13,7 @@ on:
 jobs:
   consensus-changelog:
 
-    name: Changelog sanity check
+    name: Changelogs
```

Comment:
I think the branch protection rules have to be adapted due to these renamings.

Reply:
Will do